src/styles/components/_header.scss

```diff
@@ -71,6 +71,7 @@
     &__content {
       img {
         position: absolute;
+        background-color: rgb(250,250,250);
```

Comment:
I'd prefer to use hex here instead of `rgb` to keep coloring consistent across the code base.